const handleError = require("./handleError");

module.exports = replyFirstTimer = async (ctx) => {
  try {
    const message = `‚õî *Join Our Channel And Group To Use The Botü§ñ*

‚û°Ô∏è t.me/dexhaletechs

‚û°Ô∏è t.me/dexhale

*After Joining, Click on ‚úÖJoined to continue.*
`;
    await ctx.reply(message, {
      parse_mode: "Markdown",
      disable_web_page_preview: true,
      reply_markup: {
        inline_keyboard: [
          [
            {
              text: "‚úÖ Joined",
              callback_data: "verify_join",
            },
          ],
        ],
      },
    });
  } catch (error) {
    handleError(ctx, error);
  }
};
